lib/xclarity_client/services/update_policy_management.rb

```diff
@@ -0,0 +1,74 @@
+require 'json'
+
+module XClarityClient
+  class UpdatePolicyManagement < XClarityBase
+
+    include XClarityClient::ManagementMixin
+
+    def initialize(conf)
```

Comment:
You can use the `initialize` method of the `Services::XClarityService` class

Reply:
thanks.. i have made respective update